cmake/modules/FindEGL.cmake

```diff
@@ -29,6 +29,10 @@ if(NOT TARGET EGL::EGL)
                                     VERSION_VAR EGL_VERSION)
 
   if(EGL_FOUND)
+    list(APPEND GL_INTERFACES_LIST "egl")
+    list(APPEND GL_INTERFACES_LIST "egl-pb")
```

Comment:
you should just be able to do 
```
list(APPEND GL_INTERFACES_LIST egl egl-pb)
```
You can bracket each one if you want, or just add "egl-pb") on a new line like this if you prefer for distinction

```
    list(APPEND GL_INTERFACES_LIST "egl"
                                                                 "egl-pb")
```

Reply:
updated